test/e2e/framework/pv_util.go

```diff
@@ -694,13 +726,42 @@ func newAWSClient(zone string) *ec2.EC2 {
 	return ec2.New(session.New(), cfg)
 }
 
-func createPD(zone string) (string, error) {
+func PDExists(name, zone string) (bool, error) {
 	if zone == "" {
 		zone = TestContext.CloudConfig.Zone
 	}
 
 	if TestContext.Provider == "gce" || TestContext.Provider == "gke" {
-		pdName := fmt.Sprintf("%s-%s", TestContext.Prefix, string(uuid.NewUUID()))
+		gceCloud, err := GetGCECloud()
+		if err != nil {
+			return false, err
+		}
+
+		if zone == "" && TestContext.CloudConfig.MultiZone {
+			zones, err := gceCloud.GetAllZonesFromCloudProvider()
+			if err != nil {
+				return false, err
+			}
+			zone, _ = zones.PopAny()
```

Comment:
Why are we just popping any random zone? Shouldn't we check all of them...?

Reply:
Currently all callers of this function pass in a zone anyway, so we can just return error if zone == ""